src/components/text/Link.jsx

```diff
@@ -59,7 +59,7 @@ export type LinkPropsType = {
   className?: ?string,
   inherited?: boolean,
   'aria-label'?: string,
-  onClick?: (e: KeyboardEvent | MouseEvent) => mixed,
+  onClick?: (e: MouseEvent) => mixed,
```

Comment:
Hey @katarzynatobis , I think that this has been an old approach, the proper type would be something like:
`(SyntheticMouseEvent<HTMLButtonElement | HTMLAnchorElement >) => void`,  or `(SyntheticMouseEvent<HTMLElement>) => void` (I am not sure if the former will work)

Please be sure to run `yarn js-typecheck` and see if typescript is generated correctly.

Reply:
fixed :)